website/docs/api/plugins/plugin-content-docs.mdx

```diff
@@ -34,6 +34,8 @@ Accepted fields:
 | Name | Type | Default | Description |
 | --- | --- | --- | --- |
 | `path` | `string` | `'docs'` | Path to the docs content directory on the file system, relative to site directory. |
+| `versionPath` | `string` |  `''` | Path to versioned files on filesystem, relative to site dir. |
```

Comment:
## TODO
Found that i missed changing the `versionPath` default value from `''` to `undefined`.
![image](https://user-images.githubusercontent.com/64214261/223048173-5ff2b87e-2498-4b92-8303-74e7f15b858a.png)